packages/web-components/src/pane/pane.template.ts

```diff
@@ -0,0 +1,39 @@
+import { ElementViewTemplate, html, ref } from '@microsoft/fast-element';
+import type { Pane } from './pane.js';
+
+/**
+ * The template for the Pane component.
+ * @public
+ */
+
+export function paneTemplate<T extends Pane>(): ElementViewTemplate<T> {
+  return html<T>`
+    <template
+      role="${x => (x.modal ? 'dialog' : 'complementary')}"
+      ?open="${x => x.open}"
+      ?modal="${x => x.modal}"
+      control-size="${x => x.controlSize}"
+      position="${x => x.position}"
+      focus-target="${x => x.focusTarget}"
+      aria-disabled="${x => x.ariaDisabled}"
+      aria-hidden="${x => (x.open ? 'false' : 'true')}"
+      aria-label="${x => x.ariaLabel}"
+      ?trap-focus="${x => x.trapFocus}"
+      tabindex="${x => (x.open ? '0' : '-1')}"
```

Comment:
The dialog instance (and likely the complimentary instance) should only receive focus if there are no focusable elements within it - this logic doesn't seem to account for that.